lib/compat/wordpress-6.3/script-loader.php

```diff
@@ -80,6 +80,5 @@ function( $settings ) {
 		// We must override what core is passing now.
 		$settings['__unstableResolvedAssets'] = _gutenberg_get_iframed_editor_assets();
 		return $settings;
-	},
```

Comment:
Because `gutenberg_resolve_assets_override` is removed, `_gutenberg_get_iframed_editor_assets` no longer needs to increase the priority for it to run later. It will be now the only callback for setting `__unstableResolvedAssets`. I see it was introduced https://github.com/WordPress/gutenberg/pull/49655/files#diff-d66c57d67603bf45e2664289929b228b0eb69b037553b5c35a3497182a7e17d5R84